com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/MyLanguageServerImpl.java

```diff
@@ -75,12 +75,14 @@
       TextDocumentService textService,
       WorkspaceService workspaceService,
       Provider<LanguageClient> clientProvider,
-      SettingsProvider settingsProvider) {
+      SettingsProvider settingsProvider,
+      MyTextDocumentService customTextService) {
```

Comment:
MyTextDocumentService is passed as a first parameter here

Reply:
that is TextDocumentService which is the interface provided by lsp4j and the parameter that I need to set is workspaceFolder which I added in the MyTextDocumentService, in order to get this into the didOpen method

Reply:
Is still not clear for me, better discuss in a parking lot or earlier if u need

Reply:
> I tested on my che with devfile based with this lsp and I haven't seen the dep file deleted at `didOpen`
> Moreover when I try to change the copybook name, the dep file still maintain the old value.

 I experienced the same issue because CHE doesn't close the file immediately and moreover we got a problem on VSCode with Zowe: "if your file is open by Zowe explorer and you close it and open it again you will see the same problem because I think that the IDE does not close the file immediately"